server/src/house/house.service.ts

```diff
@@ -74,9 +75,11 @@ export class HouseService {
         throw new NotFoundException(`House with ID ${houseId} not found.`);
       }
 
-      const houseMembers = house.userHouses.map(({ user }) => ({
+      const houseMembers = house.userHouses.map(({ isAdmin, user }) => ({
         id: user.id,
         name: user.name,
+        isAdmin,
```

Comment:
Now isAdmin is duplicated in house and houseMembers. It's a little bit confusing so delete `house.isAdmin` or  let's change `house.isAdmin` to `house.currentUserIsAdmin`. Please modify the related place too!

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/93/commits/d9f8f6235e1a0fcf2d53a5f85e9d4a4a33b786a2